### PR TITLE
Icon numeric feature

### PR DESCRIPTION
**Pull request by dkuntze**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

this PR expands the ability to have numbers in icon names while keeping existing functionality. It _does_ fail the regex vulnerability check at https://devina.io/redos-checker (but so does the existing). Will look at seeing if its possible to refactor to make it free of vulnerabilities by making this a tokenizer statemachine.

## Related Issues

Thanks for contributing!

### CODE REVIEW

**Review by tripodsan — CHANGES_REQUESTED** [review]
sorry, but we need to do a full scan of all content of all customers in order to judge the scope of this change. the last time, we created a CSO with exactly this kind of fix.

**Comment by dkuntze** [comment]
> sorry, but we need to do a full scan of all content of all customers in order to judge the scope of this change. the last time, we created a CSO with exactly this kind of fix.

No problem. In fact, I want that to happen.

File: test/fixtures/content/page-metadata-jsonld-error.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <meta name="twitter:card" content="summary_large_image">
     <meta name="twitter:title" content="Home | Helix Project Boilerplate">
     <meta name="twitter:image" content="https://helix-pages.com/default-meta-image.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
-    <script type="application/ld+json" data-error="error in json-ld: Unexpected non-whitespace character after JSON at position 10"></script>
+    <script type="application/ld+json" data-error="error in json-ld: Unexpected non-whitespace character after JSON at position 10 (line 1 column 11)"></script>
```
Comment (tripodsan) [inline test/fixtures/content/page-metadata-jsonld-error.html:11]: which node version are you using?

Comment (dkuntze) [reply test/fixtures/content/page-metadata-jsonld-error.html:11]: v22.6.0

**Comment by dkuntze** [comment]
moving to new PR

**Pull request closed without merging**